arxiv/taxonomy/definitions.py

```diff
@@ -308,6 +308,12 @@
     'acc-phys': {
```

Comment:
should this be in physics.acc-phys and in archive:physics?

Reply:
Yes. I'll back that out in a sec.